SSA.lean

```diff
@@ -2,17 +2,15 @@
 -- ====
 
 import SSA.Core.Framework
-import SSA.Core.WellTypedFramework
-import SSA.Core.EDSLNested
 
 
 -- Projects
 -- ========
 
 -- Eventually, all projects must be imported.
-import SSA.Projects.InstCombine.Alive
-import SSA.Projects.Tensor1D.Tensor1D
-import SSA.Projects.Tensor2D.Tensor2D
+-- import SSA.Projects.InstCombine.Alive
+-- import SSA.Projects.Tensor1D.Tensor1D
+-- import SSA.Projects.Tensor2D.Tensor2D
```

Comment:
Why are these commented?

Reply:
After deleting TSSA, These no longer compile. They will be fixed in separate PRs.